.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,6 +6,10 @@
 
 <!-- list any linked issues this pull request will close, or exclude if none -->
 
+## 📄 Documentation Requests
+
+<!-- list any issues created requesting documentation, quickstarts/samples or video content related to this PR -->
+
```

Comment:
```suggestion
## 📄 Documentation Requirements

<!-- list any documentation related iusses, quickstarts/samples or video content related to this PR -->

```

Reply:
We should call this requirement not requests, as not to confuse anyone that they shouldn't be responsible for docs.

Reply:
I phrased this as requests so that it wasn't inherently the user working on the feature/change that had to do the documentation, but fair enough